students/nico/styles.css

```diff
@@ -0,0 +1,187 @@
+
+/* ---- reset ---- */
+
+* {
+  /* border: 1px solid black; */
+}
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+  
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  
+}
+
+
+/* ---- typography ---- */
+* {
+  font-family: 'Raleway', sans-serif;
+}
+
+h3 {
+  text-align: center;
+  color: #011638;
+  font-family: 'Major Mono Display', monospace;
+}
+
+.center-box-side p {
+  height: 100%;
+  font-size: 2rem;
+  text-transform: uppercase;
+  font-weight: 700;
+  margin: 0;
+  
+}
+
+/* ---- layout ---- */
+
+.header-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between; 
+  align-items: center;
+}
+
+.nav-list-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-right: 20px;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+/* ---- components ---- */
+.nav-list-container li {
+  list-style: none;
+  margin: 5px;
+}
+
+.nav-list-container li>a {
+  text-decoration: none;
+  list-style: none;
+  color: #011638;
+  text-transform: uppercase;
+}
+
+.header-container img {
+  width: 2rem;
+  height: 2rem;
+  margin-left: 20px;
+}
+
+.section-container textarea {
+  margin-top: 10px;
+  border: 1px solid #141414;
+  width: 100%;
+}
+
+.submit-btn {
+  width: 100px;
+  background: #0D21A1;
+  color: #EEF0F2;
+  padding: 5px;
+  font-size: 1rem;
+  border-radius: 5%;
+  text-align: center;
+  position: relative;
+  left:33%;
+}
+
+
+/* ---- section ---- */
+.intro {
+  background: url("../myPersoPage/img/georgie-cobbs-467923-unsplash.jpg") no-repeat center center;
+  background-size: cover;
```

Comment:
I think that it won't be there. The path is different: "img/...." not the things before 😄

Reply:
ahah true thanks for this one 😄 !